src/TAlias.cpp

```diff
@@ -323,35 +349,49 @@ void TAlias::compileAll()
     mNeedsToBeCompiled = true;
     if( ! compileScript() )
     {
-        if( mudlet::debugMode ) {TDebug(QColor(Qt::white),QColor(Qt::red))<<"ERROR: Lua compile error. compiling script of alias:"<<mName<<"\n">>0;}
+        if( mudlet::debugMode ) {
+            TDebug( QColor(Qt::white), QColor(Qt::red) ) << "LUA ERROR: when compiling script of alias:"
```

Comment:
Well we internally call the entry area the "source editor" but the `TTrigger`/`TAction`/`TAlias`/`TKey`/`TScript`/`TTimer`/`TTrigger` items store it in the `mScript` member (oddly it isn't a member defined in the `Tree.h` template!)  We could rename it to `mSourceCode` but that would be a future PR...

_Will make change to `code`..._